### PR TITLE
RYA-100 Added Maven animal-sniffer plugin to build.

### PR DESCRIPTION
**Pull request by kchilton2**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -562,6 +562,29 @@ under the License.
...
+
+                <!-- Fail a build if it uses any Java features introduced after 1.7. -->
+                <plugin>
+                    <groupId>org.codehaus.mojo</groupId>
+                    <artifactId>animal-sniffer-maven-plugin</artifactId>
+                    <version>1.15</version>
+                    <configuration>
+                        <signature>
+                            <groupId>org.codehaus.mojo.signature</groupId>
+                            <artifactId>java17</artifactId>
```
Comment (pujav65) [inline pom.xml]: I think a later pull request upgrades rya back to java 8 (an upgrade that was lost somehow when we ported to apache).  Can you update this accordingly?  Then I will merge.

Comment (kchilton2) [inline pom.xml]: I'll update the signature to be for Java 1.8.

**Comment by kchilton2** [comment]
Updated to use Java 8 signatures.

**Comment by amihalik** [comment]
Merged.

**Pull request closed without merging**